app/assets/stylesheets/recipe.scss

```diff
@@ -3,11 +3,41 @@
   padding: 2rem;
 }
 
-.card-img-top {
+.card-img-top,
+.recipe-img {
   width: 100%;
 }
 
 .wrapper {
   text-align: center;
   padding: 2rem;
 }
+
+.ingridients {
+  font-size: 2.4rem;
+}
+
+li {
+  list-style-type: none;
+  font-size: 1.4rem;
+}
+
+.show-container {
+  text-align: center;
+}
```

Comment:
There is existing class provided by bootstrap with this styling -> `text-center`. Please use it in your view and remove this.

Info from https://getbootstrap.com/docs/4.0/utilities/text/